ui5con/index.html

```diff
@@ -105,37 +104,12 @@ <h4 class="header-item title-item">UI5con@SAP</h4>
 		    </div>
 	    </div>
    		
-   		<div class="Maincontent disclaimer divleft" id="cfp">
+   		<div class="Maincontent disclaimer divleft" id="main">
     			<div class="span12">
-    				<h2>SHOW US YOUR IDEAS! CHOOSE A MEDIUM &amp;</h2>
-				</div>
-				<div>
-					<a id="submitlink" href="mailto:openui5@sap.com?subject=Proposal for UI5con@SAP&body=Hi,%0D%0Athis is my session proposal.%0D%0A%0D%0AMy Name is:%0D%0A%0D%0A%0D%0ACompany / UI5 Background:%0D%0A%0D%0A%0D%0ASession Title:%0D%0A%0D%0A%0D%0ASession Description:%0D%0A%0D%0A%0D%0ASession Type [20min Lecture / 40min Lecture / 45min Hands-On / 30min Marketplace Booth Slot]:%0D%0A%0D%0A%0D%0A(For Hands-On only) Prerequisites for participants:%0D%0A" class="submitlink"><span>HAND IN YOUR TOPIC NOW</span><img src="resources/linkarrows_wide.png"></a>
-				</div>
-				<div class="cfpduration">
-					March 10th - April 3rd <span>(END OF DAY)</span> 2017
-				</div>
+    				<img class="ui5communityimg" src="resources/ui5community_s.jpg">
+			</div>		
     	</div>
-    	
-    	<section class="container">
```

Comment:
I already thought about that with Pavel and we will replace it with fancier info within this week. If you want to keep it anyway, that's fine by me.